addon/components/base/bs-button.js

```diff
@@ -234,6 +234,16 @@ export default Component.extend(TypeClass, SizeClass, {
    */
   value: null,
 
+  /**
+   * Controls if `onClick` action is fired concurrently. If `true` clicking button multiple times will not trigger
+   * `onClick` action if a Promise returned by previous click is not settled yet.
+   *
+   * This does not affect event bubbling.
+   *
+   * @property preventConcurrency
```

Comment:
Can you please add the `@public`, `@default` and `@type` annotations, so the API docs get generated properly!